tests/spec/core/highlight-spec.js

```diff
@@ -2,6 +2,12 @@
 describe("Core — Highlight", () => {
   afterAll(flushIframes);
 
+  it("highlights remote languages not bundled by default with ReSpec", async () => {
+    const doc = await makeRSDoc({}, "spec/core/highlight.html");
+    const span = doc.querySelector("pre.testlang span[class^=hljs]");
+    expect(span.innerText).toBe("funkyFunction");
```

Comment:
is this test enough to ensure it works? 😕

Reply:
I think so, as the matching `span` proves that it applied some highlight.

Reply:
My bad. IDK what I read!